README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
```

Comment:
This is good. I would like to see a style applied to the instructions that match the style of the game to give it a cohesive feel.